### PR TITLE
added well known and compatible logging methods

### PR DESCRIPTION
**Pull request by dtravin**

### CODE REVIEW

File: log_test.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func (s *TestSuite) TestRace1(c *C) {
 	var wg sync.WaitGroup
 	wg.Add(4)
 	start := make(chan struct{})
-	go func() { <-start; log.Err("failed"); wg.Done() }()
+	go func() { <-start; log.Error("failed"); wg.Done() }()
```
Comment (onokonem) [inline log_test.go]: I would suggest to keep the original line too

File: log_test.go [hunk 2]
```diff
@@ -73,7 +73,13 @@ func (s *TestSuite) TestRace2(c *C) {
 	var wg sync.WaitGroup
 	wg.Add(4)
 	start := make(chan struct{})
-	go func() { <-start; log := structlog.New(); log.SetDefaultKeyvals("key", 1); log.Err("failed"); wg.Done() }()
+	go func() {
```
Comment (onokonem) [inline log_test.go]: I would suggest to keep the original line too

File: log_test.go [hunk 3]
```diff
@@ -8,7 +8,7 @@ import (
 	"sync"
 	"testing"
 
-	"github.com/powerman/structlog"
+	"github.com/LiflandGaming/structlog"
```
Comment (onokonem) [inline log_test.go]: should we create a PR to the original repo?

File: log.go [hunk 4]
```diff
@@ -467,6 +472,24 @@ func (l *Logger) Printf(format string, v ...interface{}) {
...
+func (l *Logger) Infof(format string, v ...interface{}) {
+	l.log(INF, fmt.Sprintf(format, v...))
+}
+
+// Debugf works like log.Printf. Use level DBG.
+// Also output defaultKeyvals for prefixKeys/suffixKeys.
+func (l *Logger) Debugf(format string, v ...interface{}) {
+	l.log(DBG, fmt.Sprintf(format, v...))
+}
+
```
Comment (IAD) [inline log.go]: would be good to have context logger interface. In this case we need a method like WithValue(key, value)

File: log.go [hunk 5]
```diff
@@ -467,6 +472,24 @@ func (l *Logger) Printf(format string, v ...interface{}) {
...
 // Println works like log.Println. Use level INF.
```
Comment (IAD) [inline log.go]: as we have to implement this interface in applications. we can add few predefined decorators for logrus, structlog, ...

File: log_test.go (hunk 3 above)
Comment (dtravin) [inline log_test.go]: Sure, why not. Once we finalize our changes lets ask the author

File: log_test.go (hunk 1 above)
Comment (dtravin) [inline log_test.go]: will do. missed that

File: log_test.go (hunk 2 above)
Comment (dtravin) [inline log_test.go]: that is a really long line, `go fmt` autoformats

File: log.go (hunk 4 above)
Comment (dtravin) [inline log.go]: lets do it in separate PR
